src/core_plugins/kibana/public/dashboard/panel/panel.html

```diff
@@ -1,53 +1,74 @@
 <div class="panel panel-default" ng-class="{'panel--edit-mode': !isViewOnlyMode()}" ng-switch on="panel.type" ng-if="savedObj || error">
   <div class="panel-heading">
-    <span class="panel-title" title="{{::savedObj.title}}">
+    <span
+      data-test-subj="dashboardPanelTitle"
+      class="panel-title"
+      aria-label="{{:: 'Visualization panel: ' + savedObj.title }}"
```

Comment:
This could be a saved search, and once we extend panels to plugins, it might be something else entirely - e.g. input controls or monitoring.

But I don't know, maybe we call all panel types "visualizations"?  I think that might get confusing though.

I plan to refactor some of this out so that dashboard knows less about what it's rendering.  At that point, the "panelRenderer" can supply a "type" attribute, and we can use that to describe the panel type here.  In the mean time, I'll leave it up to you.  I plan on having a type attribute anyway to dynamically render the add panel tabs.